doc/project/credits.rst

```diff
@@ -1,4 +1,4 @@
-.. Note: This file is auto-generated using generate_credits.py
+.. Note: This file is auto-generated using doc/release/generate_credits.py
```

Comment:
Should `generate_credits.py` have been moved into `project` instead of `release`?
Also, this comment is auto-generated, but you haven't updated the generator to match.